src/libraries/illink-sharedframework.targets

```
@@ -14,19 +14,11 @@
       <RuntimePackILLinkArgs>$(ILLinkArgs)</RuntimePackILLinkArgs>
       <!-- update debug symbols -->
       <RuntimePackILLinkArgs>$(RuntimePackILLinkArgs) -b true</RuntimePackILLinkArgs>
-      <!-- suppress warnings with the following codes:
-           IL2009: Could not find method A in type B specified in resource C
-           IL2025: Duplicate preserve of A in B
-      -->
-      <LinkerNoWarn>IL2009;IL2025</LinkerNoWarn>
       <!-- https://github.com/dotnet/runtime/issues/40336 - need to also suppress the following on non-Windows:
            IL2008: Could not find type A specified in resource B
-           and on 32-bit builds:
-           IL2012: Could not find field 'Padding' in type 'System.Runtime.CompilerServices.RawArrayData'
       -->
       <LinkerNoWarn Condition="'$(TargetOS)' != 'windows'">$(LinkerNoWarn);IL2008</LinkerNoWarn>
```

```suggestion
      <LinkerNoWarn Condition="'$(TargetOS)' != 'windows'">IL2008</LinkerNoWarn>
```

This is the place that defines this property (see line 21 previously). No need to append a `;` to the beginning of the list :)